### DISCUSSION TITLE
Add MSC or ASC as a parent when a MSC or ASC code is added

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1216] [Ideas]
### What
- [ ] Add MSC or ASC as a parent when a MSC or ASC code is added
- [ ] Probably to be integrated at the level of the Packager Code https://github.com/openfoodfacts/robotoff/blob/5b9252d298bda2b56fdd530aee50e13112f13912/robotoff/insights/importer.py#L474
- [ ] Add the en:sustainable-seafood-msc in labels

**Comment by alexgarel** [comment]
@teolemon I'm a bit lost because the ticket description is a bit cryptic.

The scope is just to add the asc or msc label on the product right ?

(what is this notion of "parent" in the ticket ?)
